extensions/ql-vscode/src/run-queries.ts

```diff
@@ -226,6 +226,7 @@ async function convertToQlPath(filePath: string): Promise<string> {
       const fileName = path.basename(filePath);
       const fileNames = await promisify<string, string[]>(fs.readdir)(dir);
       for (const name of fileNames) {
+        // leave the locale argument empty so that the default OS locale is used.
```

Comment:
I don't think this is blocking, but I am curious: why is it more appropriate to use OS default locale instead of `env.language` here?

Reply:
Because we're looking at the filesystem, the locale of these files are always coming from the os, regardless of the locale of the running vscode instance.

I'll clarify that.